javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+window.addEventListener('beforeunload', function (event) {
+  var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+  var todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+
+  if (previousTodosJSON !== null) {
```

Comment:
This should also be outside of the function, as it is checking the data retrieved from local storage to make sure it is not null before assigning it to the `todos` variable.